tests/storage/backends_test.py

```diff
@@ -30,10 +30,11 @@
 from vdsm.storage import localFsSD
 from vdsm.storage import nfsSD
 
-from . marks import requires_sanlock
+sanlock = pytest.importorskip(
+    modname='sanlock',
+    reason="sanlock is not available")
```

Comment:
Not really needed, sanlock is always available and required. Missing sanlock is
leftover from the porting to python 3.